File: python/putData.py
import boto3
import pprint
import time
import json

with open('./config/config.json', 'r') as file:
    config = json.loads(file.read())

dynamodb = boto3.resource(
    'dynamodb',
    region_name           = 'us-east-2',
    aws_access_key_id     = config['ID'],
    aws_secret_access_key = config['KEY']
)

table = dynamodb.Table('ExampleTableTwo')

with open('./exampleData/data.json', 'r') as file:
    data = json.loads(file.read())

dataArray = data['data']

for i in dataArray:
    dataID   = i['id']
    dataFact = i['fact']
    response = table.put_item(
        Item= {
            'id'  : dataID,
            'fact': dataFact
        })
    printer = pprint.PrettyPrinter(indent=2)
    printer.pprint(response)
    time.sleep(.500)

File: node/putData.js
/**
 * Loading the table with ./exampledata/data.json
 */
'use strict'
const fs = require('fs')
const AWS = require('aws-sdk')
const {config} = require('../config/config.js')

AWS.config.update({
    region: 'us-east-2',
    accessKeyId: config.id,
    secretAccessKey: config.key
})

const docClient = new AWS.DynamoDB.DocumentClient()

let parsedData

try {
    parsedData = JSON.parse(fs.readFileSync('./exampleData/data.json', 'utf-8'))
} catch (err) {
    console.error('Unable to parse data: ', err)
}

parsedData.data.map((item) => {
    const params = {
        'TableName': 'ExampleTable',
        'Item': {
            'id': item.id,
            'fact': item.fact
        }
    }
    docClient.put(params, (error, data) => {
        error
            ? console.error('Unable to add. Error: ', JSON.stringify(error, null, 4))
            : console.log('PutItem succeeded: ', JSON.stringify(item, null, 4))
    })
})
